MiniAn/minian_main.py

```diff
@@ -177,6 +177,8 @@ def preview(minian_parameters):
     client.close()
     cluster.close()
 
+    print(mn_defs.Messages.PROCESS_DONE, flush=True)
```

Comment:
I added this function for me manly to know if the full script run or not because the print are shown in the Qtcreator application out put